src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackService.java

```diff
@@ -43,22 +47,7 @@ public CallbackService(final SecurityUtils securityUtils,
         this.defaultRetries = applicationParams.getCallbackRetries();
     }
 
-    public Optional<CallbackResponse> send(final String url,
-                                           final List<Integer> callbackRetries,
-                                           final CaseEvent caseEvent,
-                                           final CaseDetails caseDetails) {
-        return send(url, callbackRetries, caseEvent, null, caseDetails);
-    }
-
-    public Optional<CallbackResponse> send(final String url,
-                                           final List<Integer> callbackRetries,
-                                           final CaseEvent caseEvent,
-                                           final CaseDetails caseDetailsBefore,
-                                           final CaseDetails caseDetails) {
-
-        return send(url, callbackRetries, caseEvent, caseDetailsBefore, caseDetails, false);
-    }
-
+    @Retryable(value = { CallbackException.class }, maxAttempts = 3, backoff = @Backoff(delay = 1000, multiplier = 3))
```

Comment:
can we add a comment line here with the actual backoff times we get with @Backoff(delay = 1000, multiplier = 3)?

Reply:
done